### ISSUE TITLE
formation fusion360 custom

### ISSUE THREAD

**Issue by nicolasdb** [nicolasdb/nicolasdb.github.io#98]
J'ai 2 demandes en cette fin de mois pour une formation/guidance sur fusion360. 

J'aimerai 
- faire l'analyse des besoins des 2 et 
- proposer du contenu adapté en tuto existant
- définir un excercice concret pour eux
- définir x journées de guidance où je peux conseiller et guider de manière très spécifique selon les bloquages rencontrés sur leur projet.

**Comment by nicolasdb** [comment]
1. modélisation vers impression 3D et puis mise en plan 2D; 
2. modélisation pour mobilier et CAM gcode cnc.

**Comment by nicolasdb** [comment]
@zuperninja ça peut être aussi un cool truc à préparer ensemble avec ton expérience de formation. :)

**Comment by zuperninja** [comment]
Yes ! On fait ça :) Tu m'envoies leur mail et info en privé et je te montre comment je fais ça :)

**Comment by nicolasdb** [comment]
linked to #99 
@zuperninja est-ce que tu fais des articles sur le type de service que tu rend? Ou tu ajoute juste dans ton "about" ou quelque part ce que tu offre et ce que ça coute? 

Et est-ce que je met ça sur mon site ou on fixe ça sur openfab? 
On pourrait réunir les différentes offrent de mentoring par les membres sur OF mais que ça reste aussi un skill sur le portfolio. OF n'étant que la passerelle de visibilité et le lieux de prestation.
